packages/yoshi-flow-editor-runtime/src/flow-api/Base.ts

```diff
@@ -0,0 +1,31 @@
+import Experiments from '@wix/wix-experiments';
+import { ExperimentsConfig } from '../constants';
+import { ReportError } from '../types';
+import {
+  initExperimentsGetter,
+  initEmptyExperimentsGetter,
+} from '../fetchExperiments';
+
+export class FlowAPI {
+  getExperiments: () => Promise<Experiments>;
+
+  constructor({
+    experimentsConfig,
+  }: {
+    experimentsConfig: ExperimentsConfig | null;
+  }) {
+    if (experimentsConfig) {
+      this.getExperiments = initExperimentsGetter(experimentsConfig);
+    } else {
+      this.getExperiments = initEmptyExperimentsGetter();
+    }
+  }
+
+  reportError: ReportError = (error) => {
+    console.warn(
+      "You are trying to report an error, but didn't configure it in `.application.json`",
```

Comment:
```suggestion
      "You are trying to report an error, but didn't configure sentry in `.application.json`\n\nPlease read the docs for more information https://wix.github.io/yoshi/docs/editor-flow/structure-api/app-configuration/#applicationjson",
```

It would be better if we would have headers for each property in the config so we'll be able to send to specific anchor